modules/getting-started-openshift-gitops.adoc

```diff
@@ -0,0 +1,128 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/working-with-openshift-gitops.adoc
+
+[id="getting-started-with-red-hat-openshift-gitops_{context}"]
+= Getting Started with Red Hat OpenShift GitOps
+
+[id="installing-red-hat-openshift-gitops_{context}"]
+== Installing Red Hat OpenShift GitOps
+
+.Prerequisite
+
+The Red Hat product version of Argo CD is the Openshift GitOps Operator. When you install the Red Hat OpenShift GitOps operator, you get an instance of Argo CD in the 'openshift-gitops' namespace.
```

Comment:
Remove the sentence: The Red Hat product version of Argo CD is the Openshift GitOps Operator.
Suggest something like: 
* You have installed the Red Hat OpenShift GitOps operator listed in the OpenShift OperatorHub. Once installed, it is applicable to the entire cluster. When you install the Red Hat OpenShift GitOps operator, you get an instance of Argo CD in the 'openshift-gitops' namespace.

Reply:
Another reason why we need an assembly, no xrefs are allowed from modules, so prereqs with links are added to the assembly.

Reply:
Add sentence at the end about how it also adds an Argo CD icon  in the console toolbar.

Reply:
Might have to mention the precise areas where it's shown.